code/datums/wounds/burns.dm

```diff
@@ -0,0 +1,334 @@
+		return
+
+	user.visible_message("<span class='danger'>[user] carves away some of the dead flesh from [victim]'s [limb.name]!</span>", "<span class='notice'>You carve away some of the dead flesh from [victim]'s [limb.name]!</span>", ignored_mobs=victim, vision_distance=COMBAT_MESSAGE_RANGE)
+	to_chat(victim, "<span class='danger'><b>[user] crudely carves away some of the dead flesh from your [limb.name]!</b></span>")
+	limb.receive_damage(brute=I.force, wound_bonus=CANT_WOUND) // i'll be very nice and not wound for now!
+	victim.bleed(I.force) // not TOO nice though
+	mortification = max(0, mortification - (I.force * 0.05)) // 12 force scalpel removes .6 mortification
+	if(prob(60))
+		victim.emote("scream")
+
+	if(mortification > 0)
+		try_treating(I, user)
+	else
+		to_chat(user, "<span class='green'>You've hacked away enough dead flesh from [victim]'s [limb.name] to let it recover.</span>")
+		to_chat(victim, "<span class='green'>[user] finishes hacking enough dead flesh from your [limb.name] to let it recover.</span>")
+
+/datum/wound/burn/treat(obj/item/I, mob/user)
+	if(istype(I, /obj/item/stack/medical/gauze))
+		bandage(I, user)
+	else if(istype(I, /obj/item/stack/medical/ointment))
+		ointment(I, user)
+	else if(istype(I, /obj/item/stack/medical/mesh))
+		mesh(I, user)
+	else if(istype(I, /obj/item/flashlight/pen/paramedic))
+		uv(I, user)
+	else if(I.sharpness)
+		shave(I, user)
+
+/datum/wound/burn/proc/regenerate_flesh(amount)
+	flesh_healing += amount * 0.5 // 20u patch will heal 10 flesh standard
+
+// we don't even care about first degree burns, straight to second
+/datum/wound/burn/moderate
+	name = "Second Degree Burns"
+	desc = "Patient is suffering considerable burns with mild skin penetration, weakening limb integrity and increased burning sensations."
+	treat_text = "Recommended application of disinfectant and salve to affected limb, followed by bandaging."
+	examine_desc = "is badly burned and breaking out in blisters"
+	occur_text = "breaks out with violent red burns"
+	severity = WOUND_SEVERITY_MODERATE
+	damage_mulitplier_penalty = 1.25
+	threshold_minimum = 40
+	threshold_penalty = 30 // burns cause significant decrease in limb integrity compared to other wounds
+	status_effect_type = /datum/status_effect/wound/burn/moderate
+	flesh_damage = 5
+	scarring_descriptions = list("small amoeba-shaped skinmarks", "a faded streak of depressed skin")
+
+/datum/wound/burn/severe
+	name = "Third Degree Burns"
+	desc = "Patient is suffering extreme burns with full skin penetration, creating serious risk of infection and greatly reduced limb integrity."
+	treat_text = "Recommended immediate disinfection and excision of ruined skin, followed by bandaging."
+	examine_desc = "appears seriously charred, with aggressive red splotches"
+	occur_text = "chars rapidly, exposing ruined tissue and spreading angry red burns"
+	severity = WOUND_SEVERITY_SEVERE
+	damage_mulitplier_penalty = 1.5
```

Comment:
ah, I found out why I was confused earlier

you misspelled "multiplier" in this variable's name